### PR TITLE
[DUOS-1851][risk=no] link vote summary boxes to their respective slabs

### PR DESCRIPTION
**Pull request by shaemarks**
[Link to Jira ticket](https://broadworkbench.atlassian.net/browse/DUOS-1851)
Summary of changes:
- Create general scroll button that scrolls on the page to a specified location 
- Add links on labels of vote summary header to scroll to their respective slabs
- Add button on slabs to scroll to top of the page 

Note: some additional changes are shown from settling merge conflicts / overwritten changes from previous tickets. 
[DUOS-1874 Application Information details](https://broadworkbench.atlassian.net/browse/DUOS-1874)
[DUOS-1776 Member votes appearing in Chair Tab](https://broadworkbench.atlassian.net/browse/DUOS-1776)

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/components/collection_voting_slab/ResearchProposalVoteSlab.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ import {
 import VotesPieChart from '../common/VotesPieChart';
 import VoteSummaryTable from '../vote_summary_table/VoteSummaryTable';
 import CollectionAlgorithmDecision from '../CollectionAlgorithmDecision';
+import {convertLabelToKey} from "../../libs/utils";
```
Comment (rushtong) [inline src/components/collection_voting_slab/ResearchProposalVoteSlab.js:17]: Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9655076)
```suggestion
import {convertLabelToKey} from '../../libs/utils';
```

File: src/components/collection_voting_slab/ResearchProposalVoteSlab.js [hunk 2]
```diff
@@ -14,6 +14,9 @@
...
+import {ScrollToTopButton} from "../ScrollButton";
```
Comment (rushtong) [inline src/components/collection_voting_slab/ResearchProposalVoteSlab.js:18]: Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9655076)
```suggestion
import {ScrollToTopButton} from '../ScrollButton';
```

**Review by rushtong — APPROVED** [review]
This looks good, no feedback from a code perspective. One detail is that the links are active in the Application Information tab even though they don't go anywhere since the links are on the member/chair tabs. Feel free to look into it, but it should not hold up this PR.

**Review by quazi-h — APPROVED** [review]
Code changes look good and the new UI features are working correctly when I run this branch locally. I am able to click on the individual vote summaries at the top and it will bring me to the respective slab. The up arrow button on each slab also brings me back to the top. Nice work!

**Pull request merged**